src/transport/IonGasTransport.cpp

```diff
@@ -0,0 +1,388 @@
+                double xi = m_speciesCharge[i] * m_speciesCharge[i];
+                       xi *= 1.0 + pow((2 * r_alpha ),(2./3.));
+                       xi *= sqrt(alphaA_j);
+                       xi = alphaA_i / xi;
+
+                // the collision diameter
+                double K1 = 1.767;
+                double kappa = 0.095;
+                m_diam(i,j) = K1;
+                m_diam(i,j) *= pow(m_alpha[i],(1./3.)) + pow(m_alpha[j],(1./3.));
+                m_diam(i,j) /= pow((alphaA_i * alphaA_j * (1.0 + 1.0 / xi)),kappa);
+
+                // The original K2 is 0.72, but Han et al. suggested that K2 = 1.44
+                // for better fit.
+                double K2 = 1.44;
+                double epsilon = K2 * ElectronCharge * ElectronCharge;
+                epsilon *= m_speciesCharge[i] * m_speciesCharge[i];
+                epsilon *= m_alpha[j] * (1.0 + xi);
+                epsilon /= 8 * Pi * epsilon_0 * pow(m_diam(i,j),4);
+
+                if (epsilon != 0.0) {
+                    m_epsilon(i,j) = epsilon;
+                }
+
+                // Calculate dipersion coefficient and quadrupole polarizability
+                // from curve fitting if not available.
+                // Neutrals
+                if (m_disp[j] == 0.0) {
+                    m_disp[j] = exp(1.8846*log(alphaA_j)-0.4737)* 1e-50;
+                }
+                if (m_qua_polar[j] == 0.0) {
+                    m_qua_polar[j] = 2.0 * m_disp[j];
+                }
+                // Ions
+                if (m_disp[i] == 0.0) {
+                    if (m_speciesCharge[i] > 0) {
+                        m_disp[i] = exp(1.8853*log(alphaA_i)+0.2682)* 1e-50;
+                    } else {
+                        m_disp[i] = exp(3.2246*log(alphaA_i)-3.2397)* 1e-50;
+                    }
+                }
+
+                // The binary dispersion coefficient is determined by the combination rule
+                // Reference:
+                // Tang, K. T. "Dynamic polarizabilities and van der Waals coefficients."
+                // Physical Review 177.1 (1969): 108.
+                double C6 = 2.0 * m_disp[i] * m_disp[j] /
+                            (1.0/r_alpha * m_disp[i] + r_alpha * m_disp[j]);
+
+                m_gamma(i,j) = 2.0 / pow(m_speciesCharge[i],2) * C6 + m_qua_polar[j];
+                m_gamma(i,j) /= m_alpha[j] * m_diam(i,j) * m_diam(i,j);//Dimensionless
+
+                // properties are symmetric (not sure)
+                m_diam(j,i) = m_diam(i,j);
+                m_epsilon(j,i) = m_epsilon(i,j);
+                m_gamma(j,i) = m_gamma(i,j);
+            }
+        }
+    }
+}
+
+double IonGasTransport::omega11_n64(const double tstar, const double gamma)
+{
+    double logtstar = log(tstar);
+    double om11 = 0.0;
+    if (tstar < 0.01) {
+        throw CanteraError("IonGasTransport::omega11_n64(tstar, gamma)",
+                           "tstar = ", tstar," is smaller than 0.01");
```

Comment:
This syntax is not correct. The second argument should be a format string using `{}` as the substitution variables, with the following arguments being those substitutions, e.g.
```cpp
throw CanteraError("IonGasTransport::omega11_n64(tstar, gamma)",
                   "tstar = {} is smaller than 0.01", tstar);
```